### PR TITLE
Feature/567 investigate mpld3

### PR DESCRIPTION
**Pull request by nikgraf**
https://trello.com/c/m5CIoJvJ/567-investigate-mpld3

- The chart bar is now extended by reset. That's why the name changed to chart-toolbar.
- Added the latest mpld3 version including a bug fixed which already got merged.
- Replaced custom d3 charts with generated mpld3 charts.

### CODE REVIEW

File: server/src/sim/utils.py [hunk 1]
```diff
@@ -223,13 +225,25 @@ def dumpfile(variable):
         inds = argsort(printnames)
     else:
         inds = argsort(printbytes)
-    
+
     for v in inds:
         print('Variable %s is %s' % (printnames[v], printsizes[v]))
-    
+
```
Comment (Ashalynd) [inline server/src/sim/utils.py]: Is this one using globals?

Comment (Ashalynd) [inline server/src/sim/utils.py]: (fine until checkmem is not being called from any web function, I guess)

**Comment by Ashalynd** [comment]
Verified that it works on my machine. BE should be fine. Please @dmitryevseev and/or @sebastianconcept review the FE

**Comment by dmitryevseev** [comment]
I'm taking a look

File: server/src/sim/utils.py (hunk 1 above)
Comment (cliffckerr) [inline server/src/sim/utils.py]: checkmem is just a helper function, not even used on the BE, but useful for e.g. looking at where the JSON files blow out in size

**Pull request merged**